### ISSUE TITLE
Есть возможность подключить ККМ?

### ISSUE THREAD

**Issue by AlinaCuragau** [Contabilizare/c5#619]
@Nullarity Есть возможность подключить ККМ и весть сразу учет в базе? Спасибо!

**Comment by Nullarity** [comment]
Да, поддерживается печать чеков кассовых аппаратов, подключенных к системе DxPrint.

**Comment by Nullarity** [comment]
Кое-какая информация есть еще и здесь https://cont.help/d/Sale/

**Comment by AlinaCuragau** [comment]
Спасибо, буду изучать.

**Comment by AlinaCuragau** [comment]
А печать периодического отчета возможна? Или возможности по ККм зависят от самого аппарата?

**Comment by Nullarity** [comment]
Посмотрите пожалуйслта на отчет Продажи / Продажи в розницу за день. Я знаю им активно пользуются для похожей задачи. Если это не то, закидывайте задачу, будем дорабатывать.
